### ISSUE TITLE
Hidden download link from remote host

### ISSUE THREAD

**Issue by sinask** [easydigitaldownloads/easy-digital-downloads#7650]
### Explain your enhancement (please be detailed)
in your product if download link belong from another server (download server) link doesn't be encrypted and edd show true download link.
if you add feature for edd that support remote link it is going to be perfect  

scenario ::

WordPress website with edd   -> if file hosted here ->true  download link will be hidden 

WordPress website with edd   -> if file not hosted here ->example : another CPanel server with high amount of disk space -> true  download link will be shown -> and everyone can download it

**Comment by cklosowski** [comment]
We’ll have to attempt to replicate this but @sinask, have you verified that you are using the ‘forced’ download method in Downloads > Settings > File Downloads?

**Comment by sinask** [comment]
> We’ll have to attempt to replicate this but @sinask, have you verified that you are using the ‘forced’ download method in Downloads > Settings > File Downloads?

Yes i tried all options and ask around about that .finally i found out EDD doesn't support this option
